.env

```diff
@@ -7,3 +7,4 @@ VUE_APP_MS_WG_API=https://wg-api.gametactic.eu
 VUE_APP_ROOM_PRIMARY_COLOUR=#004e8c
 VUE_APP_DEFAULT_LOCALE=en
 VUE_APP_LOCALE=en
+VUE_APP_AUTH_EXPIRY_PERIOD=23.80
```

Comment:
This file should not even exist.

Reply:
it isn't a file it is a variable, I didn't want to hard code into the app incase the developers wanted to have a different refresh period

Reply:
It should also be 6. I forgot to change it

Reply:
.env doesnt variable at all. Only .env.example

Reply:
.env should not exist at all. Only .env.example

Reply:
then we need to add it to gitignore, I can do that